### DISCUSSION TITLE
HTML5 WebRTC  probléme changement de php dans le script bonjour

### DISCUSSION THREAD

**Discussion by AngeDeJade** [community/community#128106] [Programming Help] [answered]
### Select Topic Area

Product Feedback

### Body

j'ai acheter un chat html5 WebRTC a [visitchatdesign.com](https://www.visichatdesign.com/) j'ai payer sa une blinde et je me suis arnaqué le chat fonctionne mais a moitié il n'est pas fini il peux ce monté que sur centos7 en php.5.6 le souci aussi c'est que quand l'user perd sont mdp il n'as pas non plus le retour d'email lui disant de re créer un autre pass les profil ne fonctionnes pas plus quand on essaie de voir le profil de l'user après bon on peux chater a l'écris ou en cam aucun souci ce que j'aimerais pouvoir faire c'est dans un premier temps changer le php du script si je ne peux pas le jour ou les servers abandonne centos7 je ne peux plus descendre en php.5.6si vous pouvez me donner des conseil merci d'avance

**Comment by AngeDeJade** [comment]
je suis prêt a offrir le chat a celui ou celle qui saura changer le php et finir le scipt pour le moment à l'install j'ai besoin de sentora une ancienne version bien sur a modifiant quand même une ligne du script de sentora après l'install ce déroule a peux prêt normalement on va dire si le certbot passe du premier et que pm2 s'installe sans souci tout le reste fonctionne parfaitement

**Comment by gracianoandreleite** [comment]
Je peux aider à mettre à jour le script PHP vers une version plus récente, en identifiant et en corrigeant les syntaxes obsolètes et en testant le code avec PHP 7.4 ou PHP 8. Cela aidera à garantir que le chat fonctionne sur des serveurs plus modernes.

**Reply by AngeDeJade** [reply]
il faut que l'on trouve un endroit sécurisé pour que je vous envoie le script tranquillement

**Reply by AngeDeJade** [reply]
vous pouvez me joindre ici https://www.corsica-tchat.fr

**Reply by gracianoandreleite** [reply]
Je serais heureux de pouvoir aider. Cependant, il semble que le chat que vous avez suggéré rencontre quelques problèmes. Y a-t-il un autre endroit où nous pourrions communiquer plus facilement? Ou peut-être avez-vous une autre suggestion pour résoudre le problème?

**Reply by AngeDeJade** [reply]
avez vous skype

**Reply by AngeDeJade** [reply]
edelapoigne@gmail.com

**Reply by AngeDeJade** [reply]
mon adresse skype edelapoigne@live.fr

**Reply by AngeDeJade** [reply]
alors dite moi on fait quoi

**Reply by gracianoandreleite** [reply]
Je suggère que vous créiez un répertoire privé sur GitHub et que vous m'ajoutiez en tant que collaborateur.

**Reply by AngeDeJade** [reply]
je suis désolé mais je en sais pas comment on fait ce genre de chose ici sur ghitub

**Comment by AngeDeJade** [comment]
bonjour ok j ai un server je peux vous envoyer le scripts si vous en avais
besoin

Cordialement

Le mar. 11 juin 2024 à 13:40, Graciano André ***@***.***> a
écrit :

**Comment by LiteBrite82** [comment, marked as answer]
Hi @AngeDeJade , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
